src/pages/fr/core-concepts/sharing-state.md

```diff
@@ -0,0 +1,699 @@
+
+Pourtant, il existe un certain nombre d'alternatives que vous pouvez explorer. Ceux-ci incluent :
+- Les [Stores intégrés de Svelte](https://svelte.dev/tutorial/writable-stores)
+- [Les signaux de SolidJS](https://www.solidjs.com/docs/latest) en dehors du contexte d'un composant
+- [L'envoi d'événements personnalisés du navigateur](https://developer.mozilla.org/fr/docs/Web/Events/Creating_and_triggering_events) entre les composants
+
+:::note[FAQ]
+
+<details>
+<summary>**🙋 Puis-je utiliser Nano Stores dans des fichiers `.astro` ou d'autres composants côté serveur ?**</summary>
+
+Nano Stores _peut_ être importé, écrit et lu dans les composants côté serveur, **mais nous ne le recommandons pas !** Cela est dû à quelques restrictions :
+- L'écriture d'un store à partir d'un fichier `.astro` ou [composant non hydraté](/fr/core-concepts/framework-components/#hydratation-des-composants-interactifs) n'affectera _pas_ la valeur reçue par les [composants côté client](/fr/reference/directives-reference/#client-directives).
+- Vous ne pouvez pas transmettre un Nano Store en tant que `prop` aux composants côté client.
+- Vous ne pouvez pas vous abonner aux mises à jour du store à partir d'un fichier `.astro`, puisque les composants Astro ne se rafraîchissent pas. 
+
+Si vous comprenez ces restrictions et que vous trouvez toujours un cas d'utilisation, vous pouvez essayer Nano Stores ! N'oubliez pas que les Nano Stores sont conçus pour être réactifs aux changements sur le **client** spécifiquement.
+
+</details>
+
+<details>
+<summary>**🙋 En quoi les Stores de Svelte sont-ils différents de Nano Stores?**</summary>
+
+**Nano Stores et les [Stores de Svelte](https://svelte.dev/tutorial/writable-stores) sont très similaires!** En effet, [Nano Stores vous permettent d'utiliser le même raccourci `$`](https://github.com/nanostores/nanostores#svelte) pour les abonnements que vous pourriez utiliser avec Svelte stores.
+
+Si vous voulez éviter les bibliothèques tierces, [Svelte stores](https://svelte.dev/tutorial/writable-stores) sont à eux seuls un excellent outil de communication inter-îles. Néanmoins, vous préférerez peut-être Nano Stores si a) vous aimez leurs modules complémentaires pour les [objets](https://github.com/nanostores/nanostores#maps) et [l'état asynchrone](https://github.com/nanostores/nanostores#lazy-stores), ou b) vous souhaitez communiquer entre Svelte et d'autres frameworks d'interface utilisateur comme Preact ou Vue.
+</details>
+
+<details>
+<summary>**🙋 Comment Solid signals est-il différent de Nano Stores?**</summary>
+
+Si vous utilisez Solid depuis un certain temps, vous avez peut-être essayé de déplacer [signals](https://www.solidjs.com/docs/latest#createsignal) ou [stores](https://www.solidjs.com/docs/latest#createstore) en dehors de vos composants. C'est un excellent moyen de partager l'état entre les Solid Islands ! Essayez d'exporter des signaux à partir d'un fichier partagé :
```

Comment:
Make changes suggested in https://github.com/withastro/docs/pull/1985#discussion_r1014531755

```suggestion
Si vous utilisez Solid depuis un certain temps, vous avez peut-être essayé de déplacer les [signals](https://www.solidjs.com/docs/latest#createsignal) ou [stores](https://www.solidjs.com/docs/latest#createstore) en dehors de vos composants. C'est un excellent moyen de partager l'état entre les Îles de composants Solid ! Essayez d'exporter des "signals" à partir d'un fichier partagé :
```